ports/nvpw/port.json

```
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "nvpw.zip",
+  "items": [
+    "NOT VERY PRO WRESTLING.sh",
+    "nvpw/"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "NOT VERY PRO WRESTLING ",
```

Superfluous extra space in title.
```suggestion
    "title": "NOT VERY PRO WRESTLING",
```